src/monitor/main_monitor.cpp

```diff
@@ -88,12 +88,6 @@ otf2::definition::metric_class MainMonitor::generate_metric_class()
                                           otf2::common::metric_mode::accumulated_start,
                                           otf2::common::type::Double, "#"));
     }
-    c.add_member(trace_.metric_member("time_enabled", "time event active",
-                                      otf2::common::metric_mode::accumulated_start,
-                                      otf2::common::type::uint64, "ns"));
-    c.add_member(trace_.metric_member("time_running", "time event on CPU",
-                                      otf2::common::metric_mode::accumulated_start,
-                                      otf2::common::type::uint64, "ns"));
```

Comment:
Are you sure we want to remove `time_enabled` and `time_running` by default? These aren't really metrics per se, but their values are updated from the kernel buffer every time the metric events are read. I think it would make sense to stop recording them if there's no metric events being recorded, not just outright disable them.

Reply:
What is the point of those metrics, when you're not recording any metrics? What would their values even be?

The correct behavior:
- [ ] if there are any perf metrics recorded, add those pseudo metrics
- [ ] if there are not metrics recorded, either the requested none or requested invalid metrics, then don't record them

The same goes for the metric leader.